packages/docs/pages/flex.tsx

```diff
@@ -24,153 +24,225 @@ const ExampleBox: React.FC<{ vertical?: boolean }> = ({ children, vertical }) =>
+                    items to grow relative to the flex container's width.
+                  </Text>
+
+                  <CodePreview scope={{ ExampleBox }}>
+                    {/* jsx-to-string:start */}
+                    <Flex>
+                      <FlexItem flexGrow={2} flexOrder={2}>
+                        <ExampleBox>Item 1</ExampleBox>
+                      </FlexItem>
+                      <FlexItem flexOrder={4}>
+                        <ExampleBox>Item 2</ExampleBox>
+                      </FlexItem>
+                      <FlexItem flexGrow={4} flexOrder={1}>
+                        <ExampleBox>Item 3</ExampleBox>
+                      </FlexItem>
+                      <FlexItem flexGrow={1} flexOrder={3}>
+                        <ExampleBox>Item 4</ExampleBox>
+                      </FlexItem>
+                    </Flex>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </>
+              ),
+            },
+          ]}
+        />
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <ContentRoutingTabs
+          id="props"
+          routes={[
+            {
+              id: 'flex',
+              title: 'Flex',
+              render: () => (
+                <FlexPropTable
+                  renderPanel={false}
+                  inheritedProps={
+                    <>
+                      <BoxPropTable collapsible />
+                      <DisplayPropTable collapsible />
+                      <MarginPropTable collapsible />
+                      <PaddingPropTable collapsible />
+                    </>
+                  }
+                />
+              ),
+            },
+            {
+              id: 'flex-item',
+              title: 'FlexItem',
+              render: () => (
+                <FlexItemPropTable
+                  renderPanel={false}
+                  inheritedProps={
+                    <>
+                      <BoxPropTable collapsible />
+                      <DisplayPropTable collapsible />
+                      <MarginPropTable collapsible />
+                      <PaddingPropTable collapsible />
+                    </>
+                  }
+                />
+              ),
+            },
+          ]}
+        />
+      </Panel>
 
-      <PageNavigation items={items} />
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={[
+            <>
+              Use <Code primary>Flex</Code> when focusing on content flow.
+            </>,
+            <>
+              Use <Code primary>FlexItem</Code> when children need specific flex properties.
+            </>,
+            <>
+              Use <Link href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">CSS-Tricks complete guide</Link>{' '}
+              to <Code primary>Flexbox</Code>.
```

Comment:
I don't think we need to wrap it in a `<Code>` element when it's referring to the idea of "flexbox".
```suggestion
              to flexbox.
```